db/constants.conf

```diff
@@ -3656,6 +3656,13 @@ constants_db: {
 	NPC_NAME_HIDDEN:  2
 	NPC_NAME_UNIQUE:  3
 	NPC_MAP:         4
+	
+	comment__: "getcharid"
+	ID_CHAR:         0
+	ID_PARTY:        1
+	ID_GUILD:        2
+	ID_ACCOUNT:      3
+	ID_BATTLEGROUND: 4
```

Comment:
I would suggest to make it CHAR_PARTY_ID and so.
after this change PR will be merged.